edge-modules/SimulatedTemperatureSensor/src/Program.cs

```diff
@@ -54,7 +54,7 @@ static async Task<int> MainAsync()
                 HumidityPercent = configuration.GetValue("ambientHumidity", 25)
             };
 
-            TransportType transportType = configuration.GetValue("ClientTransportType", TransportType.Amqp_Tcp_Only);
+            TransportType transportType = configuration.GetValue("ClientTransportType", TransportType.Mqtt_Tcp_Only);
```

Comment:
>TransportType transportType = configuration.GetValue("ClientTransportType", TransportType.Mqtt_Tcp_Only); [](start = 12, length = 105)

We should check if the transport protocol is set on deployment to AMQP. Looking into it. If it's set there, this change won't have any effect.

Reply:
Right, this just changes the default

Reply:
Ok. Just checked we are not setting that by default.